dll/win32/shell32/shlfileop.cpp

```diff
@@ -1615,11 +1615,17 @@ static HRESULT delete_files(FILE_OPERATION *op, const FILE_LIST *flFrom)
         && TRASH_CanTrashFile(flFrom->feFiles[0].szFullPath);
 
     if (!(op->req->fFlags & FOF_NOCONFIRMATION) || (!bTrash && op->req->fFlags & FOF_WANTNUKEWARNING))
-        if (!confirm_delete_list(op->req->hwnd, op->req->fFlags, bTrash, flFrom))
+    {
+        DWORD dwConfirmDelete;
+        DWORD dwSize = sizeof(DWORD);
+        
+        LSTATUS Result = RegGetValueW(HKEY_CURRENT_USER, L"SOFTWARE\\Microsoft\\Windows\\CurrentVersion\\Policies\\Explore", L"ConfirmFileDelete", RRF_RT_DWORD, NULL, &dwConfirmDelete, &dwSize);
```

Comment:
And here as well.

Reply:
Thanks for pointing it up, i  copied that once to the code, and then i copied it from there, so it's logicall that's wrong in all the changes, i will update the commit

Reply:
Done